rktlet/rktlet.go

```diff
@@ -51,7 +51,6 @@ func New(config *Config) (ContainerAndImageService, error) {
 
 	rktCli := cli.NewRktCLI(config.RktPath, execer, cli.CLIConfig{
 		InsecureOptions: []string{"image", "ondisk"},
-		Dir:             config.RktDatadir,
```

Comment:
Sorry, maybe I wasn't clear. What I meant is that you didn't need the *change* (it was just moving a line to other spot), not that you can remove `Dir`. Please add it back like it is on master.